lib/api/objectPut.js

```diff
@@ -119,7 +120,7 @@ function _storeIt(bucketName, objectKey, objMD, authInfo, canonicalID,
                                 newByteLength: size,
                                 oldByteLength: prevContentLen,
                             });
-                            return callback(null, contentMD5, prevContentLen);
```

Comment:
I think Utapi uses this for some metrics, right @bennettbuchanan? So we still want to calculate it. Originally it was passed to the callbacks for Utapi because utapi logging used to happen in the routes. Then when the refactoring happened to put the pushMetric calls in the api methods, we forgot to remove it from the callbacks.